test/UnitTest.cs

```diff
@@ -384,6 +384,67 @@ public void TestTpchDistributed()
             }
         }
 
+        [TestMethod]
```

Comment:
why do we need an extra test for this fix?

Reply:
I delete the unittest. q20 of TCP-H is more complex for debugging. So I construct a easy one for this issue.
I have renamed the TestForIssue72 to TestSubqueryInclusiveCost and move it to class Subquery in UnitTest.cs.

Reply:
I am not sure I got it - the original tpch test already have q20. Why you introduce TestSubqueryInclusiveCost() just to test q20 again?

Reply:
The query result you quoted  in the issue description is the  result  of  q20. I test q20 for emphasis the issue of bug in filter subquery.  If you think it is unnecessary to creat a unit test  to test q20 again, i will delete it.

Reply:
The query result you quoted  in the issue description is the  result  of  q20. I test q20 for emphasis the issue of bug in filter subquery.  If you think it is unnecessary to creat a unit test  to test q20 again, i will delete it.

Reply:
The query result you quoted  in the issue description is the  result  of  q20. I test q20 for emphasis the issue of bug in filter subquery.  If you think it is unnecessary to creat a unit test  to test q20 again, i will delete it.

Reply:
please remove this unittest as it is redundant.